### PR TITLE
(PC-12271) feat(identityCheck): add IdentityCheckPending

### PR DESCRIPTION
**Pull request by Lucasbeneston**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12271

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Added a **screenshot** for UI tickets.
- [x] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|   Capture d’écran 2021-12-13 à 17 18 17  |         |                 |   Capture d’écran 2021-12-13 à 17 20 06 |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md

### CODE REVIEW

File: src/features/identityCheck/api.ts [hunk 1]
```diff
@@ -14,11 +17,26 @@ export const REDIRECT_URL_UBBLE = `${WEBAPP_V2_URL}/verification-identite/fin`
...
     MutationKeys.IDENTIFICATION_URL,
-    () => api.postnativev1ubbleIdentification({ redirectUrl: REDIRECT_URL_UBBLE }),
-    { onSuccess: (data) => setIdentificationUrl(data.identificationUrl) }
+    async () => {
+      try {
+        const data = await api.postnativev1ubbleIdentification({ redirectUrl: REDIRECT_URL_UBBLE })
+        setIdentificationUrl(data.identificationUrl)
+      } catch (error) {
+        if ((error as ApiError)?.content.code === 'IDCHECK_ALREADY_PROCESSED') {
+          setIdentificationUrl(undefined)
```
Comment (antoinewg) [inline src/features/identityCheck/api.ts:30]: tu peux le factoriser et même l'enlever vu que c'est undefined par défaut

**Review by antoinewg — APPROVED** [review]

File: src/features/identityCheck/api.ts [hunk 2]
```diff
@@ -14,11 +17,26 @@ export const REDIRECT_URL_UBBLE = `${WEBAPP_V2_URL}/verification-identite/fin`
 
 export function useIdentificationUrl() {
   const [identificationUrl, setIdentificationUrl] = useState<string | undefined>()
-  const { mutate: requestIdentificationUrl } = useMutation<IdentificationSessionResponse, void>(
+  const { navigate } = useNavigation<UseNavigationType>()
+
+  const { mutate: requestIdentificationUrl } = useMutation(
```
Comment (anoukhello) [inline src/features/identityCheck/api.ts:22]:
```suggestion
  const { mutate: postIdentificationUrl } = useMutation(
```

**Review by anoukhello — APPROVED** [review]

**Review by antoinewg — APPROVED** [review]

File: src/features/identityCheck/api.ts (hunk 2 above)
Comment (antoinewg) [reply src/features/identityCheck/api.ts]: désolé mais je suis pas convaincu par cette suggestion

Comment (antoinewg) [reply src/features/identityCheck/api.ts]: pas entièrement convaincu

**Pull request merged**